.golangci.yml

```diff
@@ -43,7 +43,6 @@ linters:
   - paralleltest
   - gomnd
   - goerr113
-  - gochecknoglobals
```

Comment:
is this not removing the linter? or am I missing something very obvious 😕

Reply:
All linters are **enabled** by default and we have a list that disables them. 

https://github.com/form3tech-oss/f1/blob/master/.golangci.yml#L42-L51